tests/test_create_conference.py

```diff
@@ -0,0 +1,339 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import os
+import datetime
+import openreview
+import pytest
+import time
+import json
+
+os.environ["OPENREVIEW_USERNAME"] = "OpenReview.net"
+os.environ["OPENREVIEW_PASSWORD"] = "1234"
+
+class TestConference():
+    @pytest.fixture(scope="class")
+    def conference(self, client):
+        pc_client=openreview.Client(username='pc@neurips.cc', password='1234')
+        request_form=client.get_notes(invitation='openreview.net/Support/-/Request_Form', sort='tmdate')[0]
+
+        conference=openreview.helpers.get_conference(pc_client, request_form.id)
+        return conference
+
+    def test_create_client(self, client, openreview_client):
+        # Test connectivity to the API
+
+        client = openreview.Client()
+        assert client
+        assert client.token
+        assert client.profile
+        assert '~Super_User1' == client.profile.id
+
+        openreview_client = openreview.Client()
+        assert openreview_client
+        assert openreview_client.token
+        assert openreview_client.profile
+        assert '~Super_User1' == openreview_client.profile.id
+
+    def test_create_groups(self, client, helpers):
+        # Post test data groups to the API
+
+        def post_profiles(data):
```

Comment:
instead of creating the profiles, can you use `helpers.create_user`?

Reply:
The `post_profiles` function calls `helpers.create_user` after reading the profile data from the JSON files

Reply:
but create_user already creates the profile. It calls to the signup endpoint and active the profile.

Reply:
Right, that happens here:
https://github.com/openreview/openreview-expertise/blob/a2e011a580d294d1714766726b4ba77a73ac0490/tests/test_create_conference.py#L44-L47

Some of the tests reference profile IDs that end in a number greater than 1, so this function checks if that's the case and renames the profile

Reply:
Could you just use all different name so all the tilde ids end with 1?

Reply:
you can use different names for each test or have a single function to be called before all the tests that create all the profiles for all the tests.